v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -65,6 +61,23 @@ def get_ht(
         return module.get_ht(path, reference_genome, dataset_type)
 
 
+class ReferenceDataset(BaseReferenceDataset, str, Enum):
+    cadd = 'cadd'
+    clinvar = 'clinvar'
+    hgmd = 'hgmd'
+
+
+class ReferenceDatasetQuery(BaseReferenceDataset, str, Enum):
```

Comment:
I ended up keeping the "Query" pattern because it doesn't make sense for us to download `clinvar` or `gnomad` twice to generate these.   It's much more lightweight though.